dataservice/api/family_relationship/resources.py

```diff
@@ -38,7 +39,18 @@ def get(self, filter_params, after, limit):
         # Get study id and remove from model filter params
         study_id = filter_params.pop('study_id', None)
 
-        q = FamilyRelationship.query.filter_by(**filter_params)
+        # Modifies query to get all relationships given a participant's kf_id
+        participant_id = filter_params.get('_all', None)
+        if participant_id:
+            q = FamilyRelationship.query_all_relationships(participant_id)
+            filter_params.pop('participant_id', None)
+            filter_params.pop('relative_id', None)
+            filter_params.pop('_all', None)
```

Comment:
Maybe this could be a boolean field `?both=`?
If true, will specify that you want the participant AND the relative relations given the participant OR the relative.
If false, will specify that you want the participant OR the relative relations given the relative OR participant.